Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1177,12 +1177,15 @@ extension BrowserViewController: TabManagerDelegate {
             }
         }
 
-        toolbar?.updateBackStatus(selected?.canGoBack ?? false)
-        toolbar?.updateForwardStatus(selected?.canGoForward ?? false)
-        toolbar?.updateReloadStatus(selected?.loading ?? false)
+        if let tab = tabManager.selectedTab,
```

Comment:
No need for this line: we already have the selected tab from the `selected` argument given to this function.